crypto/ec/ecx_meth.c

```diff
@@ -346,10 +517,12 @@ static int ecd_item_verify(EVP_MD_CTX *ctx, const ASN1_ITEM *it, void *asn,
 {
     const ASN1_OBJECT *obj;
     int ptype;
+    int nid;
 
     X509_ALGOR_get0(&obj, &ptype, NULL, sigalg);
-    /* Sanity check: make sure it is ED25519 with absent parameters */
-    if (OBJ_obj2nid(obj) != NID_ED25519 || ptype != V_ASN1_UNDEF) {
+    nid = OBJ_obj2nid(obj);
+    /* Sanity check: make sure it is ED25519/ED448 with absent parameters */
```

Comment:
I think you should put the comment before the two assignments.  Or put a blank line before the comment.

Reply:
Done.